### DISCUSSION TITLE
From our hearts to yours 💘 [Pages Mini Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#108339] [Other Feature Feedback, Questions, & Ideas]
![Pages Space-themed check-in header graphic with two octocats standing on the moon in front of the earth](https://github.com/community/community/assets/92824518/25aa9c5d-f8ae-4b2f-98eb-b895e5343d19)

Happy Valentine’s Day! We’ve got our usual love for our members, plus we’ve got you covered for GitHubby ways to say I love you today. 
### 📄 Pros in Pages: A round of applause for these community citizens 👏
* @nugrezo: Figured out their [problem publishing their page](https://github.com/orgs/community/discussions/101515) all on their own. We love a self-solve and anyone who comes back to the community to share their knowledge. 
* @saibushifu: After some back and forth, they were able to figure out what was causing the OP’s 404 error. [See them in action[(https://github.com/orgs/community/discussions/90524) - thanks for persevering and assisting. 

###💡Inspiration Station: Looking for a last minute idea for your Valentine? We’ve got some projects to help say I love you to a special someone. 
* **Valentine’s Website**: Spread the love this holiday. Check out  [A valentine website for your bae](https://github.com/sojijr/valentine-website) from @sojijr.  
This project gives you a template to use and edit for your valentine website instead of building one from scratch
* **Will You Be My Valentine?**: @saurabhnemade built a simple application to ask your sweetheart to be your valentine. [Get started with this project](https://github.com/saurabhnemade/will-you-be-my-valentine). 

### 📣 Announcements: The latest updates in the Community 🆕
* **Achievements will no longer be available in the Community**:  In response to this fast growing issue and comments from members around the galaxy brain achievement, we have made the decision to turn off the ability to earn Achievements in our Community. [Read more for details](https://github.com/orgs/community/discussions/106536). 

No such thing as too much love - shoutout an awesome community member below to keep spreading it ❣️.

**Comment by Ariffazeel99** [comment]
interesting

**Comment by queenofcorgis** [comment]
More interesting content is here in[ March's Pages Mini check-in](https://github.com/orgs/community/discussions/112482)
